pretty_printers_tests/tests/btree.rs

```diff
@@ -24,5 +25,9 @@ fn main() {
     for i in 0..15 {
         btree_map.insert(i, i);
     }
+
+    let mut zst_btree_map: BTreeMap<(), ()> = BTreeMap::new();
```

Comment:
Maybe it worth to add the same test for `BTreeSet` as well?

Reply:
Added